### PR TITLE
Creating the Anvil Class Engine

### PR DESCRIPTION
**Pull request by Zitchas**
This engine is intended to be a smaller and more compact engine unit for the Tern (a Remnant drone), in order to get both the drone and its available space down.

This engine is a combination thruster-steering unit that takes up approximately 75% of the space of a crucible-class thruster & steering unit combined (rounded up), and provides roughly 75% of the thrust & steering (rounded down). As such, the intention is that when compared to other thrusters or steering, it should be slightly less space efficient than other Remnant engines. (although only slightly)

Note: Due to the gascraft archon dodging mission specs that are intended to make using a cloaking device to bypass that mission impossible to beginner players, this PR also makes the following changes to the gascraft:
 - Replaces the Crucible-Class thruster & steering with a Anvil-Class engine
 - Adjusts the mass & drag of the gascraft to keep the Max Speed and acceleration as close to the original as possible. (<1% variation).
- Removes outfit space & engine space to reflect the smaller propulsion system.

Link to zip file on google drive with graphical assets: 
https://drive.google.com/open?id=1uWjhTjSYm7Lm9DeMKYBdIbnFTIc4VoxL

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [X] I uploaded the necessary image, blend, and texture assets here: [link to assets](https://drive.google.com/open?id=1uWjhTjSYm7Lm9DeMKYBdIbnFTIc4VoxL)
 - [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: [PR link](https://github.com/endless-sky/endless-sky-high-dpi/pull/93)

### CODE REVIEW

**Comment by Zitchas** [comment]
By popular request, the obscure forging term "Cassette" has been replaced with the more commonly known term "Anvil".

File: data/remnant outfits.txt [hunk 1]
```diff
@@ -359,11 +359,31 @@ outfit "Salvage Scanner"
...
+	licenses
+		Remnant
+	"cost" 290000
+	thumbnail "outfit/tiny remnant engine"
+	"mass" 26
+	"outfit space" -26
+	"engine capacity" -26
+	"thrust" 13.5
+	"thrusting energy" 2.65
+	"thrusting heat" 2.5
```
Comment (Amazinite) [inline data/remnant outfits.txt]: The thrust energy and heat shouldn't be exactly the same as the Crucible-Class if the thrust is less. I'd suggest 2.05 energy and 1.95 heat.

File: data/remnant outfits.txt [hunk 2]
```diff
@@ -359,11 +359,31 @@ outfit "Salvage Scanner"
...
+	"turn" 336.0
+	"turning energy" 1.32
+	"turning heat" 1.4
```
Comment (Amazinite) [inline data/remnant outfits.txt]: Again, the turning energy and heat shouldn't match the Crucible-Class if the turn is lesser. I'd suggest 1.05 energy and 1.1 heat.

File: data/remnant outfits.txt [hunk 3]
```diff
@@ -359,11 +359,31 @@ outfit "Salvage Scanner"
...
+		"frame rate" 5
+	"flare sound" "plasma tiny"
+	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."
+
 outfit "Crucible-Class Thruster"
 	category "Engines"
 	licenses
 		Remnant
-	"cost" 200000
+	"cost" 190000
```
Comment (Amazinite) [inline data/remnant outfits.txt]: Why change the Crucible-Class thrusters?

File: data/remnant ships.txt [hunk 4]
```diff
@@ -209,8 +209,7 @@ ship "Gascraft"
 		"Emergency Ramscoop"
 		"Quantum Key Stone"
 
-		"Crucible-Class Thruster"
-		"Crucible-Class Steering"
+		"Cassette-Class Engine"
```
Comment (Amazinite) [inline data/remnant ships.txt]: You forgot to rename the outfit here and in the outfitter.

File: data/remnant outfits.txt (hunk 2 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: OK. Changed

File: data/remnant outfits.txt (hunk 1 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: Alright. Changed

File: data/remnant outfits.txt (hunk 3 above)
Comment (Zitchas) [reply data/remnant outfits.txt]: Sorry, that was accidental. Reverted.

File: data/remnant ships.txt (hunk 4 above)
Comment (Zitchas) [reply data/remnant ships.txt]: Thanks! Fixed.

File: data/remnant outfits.txt [hunk 5]
```diff
@@ -359,6 +359,26 @@ outfit "Salvage Scanner"
...
+	"thrust" 13.5
+	"thrusting energy" 2.05
+	"thrusting heat" 1.95
+	"turn" 336.0
+	"turning energy" 1.05
+	"turning heat" 1.1
+	"flare sprite" "effect/remnant flare/small"
+		"frame rate" 5
+	"flare sound" "plasma tiny"
+	description "This tiny set of engines were designed specifically for craft so small that they to combine both thrusters and steering into one unit."
```
Comment (Amazinite) [inline data/remnant outfits.txt]: `that they to combine` Seems like you messed up here.
How about `"This tiny set of engines were designed specifically for craft so small that they're unable to fit a normal set of engines, leading to both thrusters and steering being combined into one unit."`

Comment (Pointedstick) [inline data/remnant outfits.txt]:
```suggestion
	description "This tiny set of engines was designed specifically for craft so small that it combines both thrusters and steering into one unit."
	
("set of engines" is a singular noun)
```

Comment (Zitchas) [reply data/remnant outfits.txt]: Fixed by Pointedstick's change.

**Pull request merged**